From 98d17450cd4b008310351616e8a1897544713ffe Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Milan=20P=2E=20Stani=C4=87?= <mps@arvanta.net>
Date: Mon, 18 Sep 2023 10:32:29 +0200
Subject: [PATCH] starfive: visionfive2: add mmc0 and nvme boot targets
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

boot from SDIO3.0 (mmc sdcard) first if it is plugged.
If mmc is not plugged try to boot from emmc if it is plugged.
If emmc is not plugged then try to boot from nvme.

Signed-off-by: Milan P. StaniÄ‡ <mps@arvanta.net>
Reviewed-by: Leo Yu-Chi Liang <ycliang@andestech.com>
---
 include/configs/starfive-visionfive2.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/starfive-visionfive2.h b/include/configs/starfive-visionfive2.h
index 6fa7f16ddc..ff43113f24 100644
--- a/include/configs/starfive-visionfive2.h
+++ b/include/configs/starfive-visionfive2.h
@@ -20,6 +20,7 @@
 #define BOOT_TARGET_DEVICES(func) \
 	func(NVME, nvme, 0) \
 	func(USB, usb, 0) \
+	func(MMC, mmc, 0) \
 	func(MMC, mmc, 1) \
 	func(DHCP, dhcp, na)
 
-- 
2.42.0

